base/strings/basic.jl

```diff
@@ -239,11 +239,24 @@ end
 prevind(s::DirectIndexString, i::Integer) = Int(i)-1
 nextind(s::DirectIndexString, i::Integer) = Int(i)+1
 
+function prevind(s::DirectIndexString, i::Integer, nchar::Integer)
+    nchar > 0 || throw(ArgumentError("nchar must be greater than 0"))
+    Int(i)-nchar
+end
+
+function nextind(s::DirectIndexString, i::Integer, nchar::Integer)
+    nchar > 0 || throw(ArgumentError("nchar must be greater than 0"))
+    Int(i)+nchar
+end
+
+
 """
     prevind(str::AbstractString, i::Integer)
+    prevind(str::AbstractString, i::Integer, nchar::Integer)
```

Comment:
Just use a single line with `nchar=1`. Then below you can simply say that the function goes `nchar` characters, implementation details do not matter.

Reply:
fixed